tests/ui_tests/test_save_configuration_button.py

```diff
@@ -0,0 +1,326 @@
+
+    return pytest.mark.parametrize("bundle_content", params, ids=ids, indirect=True)
+
+
+@pytest.fixture()
+def bundle_content(request, tmp_path):
+    struct_filename = "struct_conf.yaml"
+    struct_path_src = os.path.join(get_data_dir(__file__), struct_filename)
+    struct_path_dest = os.path.join(tmp_path, struct_filename)
+    shutil.copyfile(
+        struct_path_src,
+        struct_path_dest,
+    )
+
+    bundle_filename = "config.yaml"
+    bundle_path = os.path.join(tmp_path, bundle_filename)
+    with allure.step("Dump YAML config to file"):
+        with open(bundle_path, "w") as stream:
+            yaml.dump(request.param["bundle"], stream, sort_keys=False)
+            allure.attach.file(
+                bundle_path,
+                name=bundle_filename,
+                attachment_type=allure.attachment_type.YAML,
+            )
+
+    yield request.param["opts"], tmp_path
+
+    os.remove(bundle_path)
+    os.remove(struct_path_dest)
+
+
+@pytest.fixture()
+def bundle(bundle_content, sdk_client_fs: ADCMClient) -> Bundle:
+    """Assume request.param to be path to the bundle"""
+    _, bundle_path = bundle_content
+    return sdk_client_fs.upload_from_fs(bundle_path)
+
+
+@pytest.fixture()
+def cluster(bundle: Bundle) -> Cluster:
+    return bundle.cluster_create(name=CLUSTER_NAME)
+
+
+@pytest.fixture()
+def cluster_config_page(app_fs, cluster: Cluster, login_to_adcm):
+    return Configuration(
+        app_fs.driver,
+        "{}/cluster/{}/config".format(
+            app_fs.adcm.url, cluster.cluster_id
+        )
+    )
+
+
+@pytest.fixture()
+def service(cluster: Cluster, sdk_client_fs: ADCMClient) -> Service:
+    cluster.service_add(name=SERVICE_NAME)
+    return cluster.service(name=SERVICE_NAME)
+
+
+@pytest.fixture()
+def service_config_page(app_fs, service: Service, login_to_adcm) -> Configuration:
+    return Configuration(
+        app_fs.driver,
+        "{}/cluster/{}/service/{}/config".format(
+            app_fs.adcm.url, service.cluster_id, service.id
+        ),
+    )
+
+
+@pytest.fixture()
+def provider(bundle: Bundle) -> Provider:
+    return bundle.provider_create(name=PROVIDER_NAME + random_string())
+
+
+@pytest.fixture()
+def provider_config_page(app_fs, provider: Provider, login_to_adcm) -> Configuration:
+    return Configuration(
+        app_fs.driver,
+        "{}/provider/{}/config".format(
+            app_fs.adcm.url, provider.provider_id
+        ),
+    )
+
+
+@pytest.fixture()
+def host(provider: Provider) -> Host:
+    return provider.host_create(fqdn=f"{HOST_NAME}_{random_string()}")
+
+
+@pytest.fixture()
+def host_config_page(app_fs, host: Host, login_to_adcm) -> Configuration:
+    return Configuration(
+        app_fs.driver,
+        "{}/host/{}/config".format(app_fs.adcm.url, host.id),
+    )
+
+
+def _get_field_input(field):
+    return field.find_element(*Common.mat_input_element)
+
+
+def _get_field_checkbox(field):
+    return field.find_element(*Common.mat_checkbox)
```

Comment:
Remove this functions to BasePage module

Reply:
Moved to Configuration